schema/va-spec/base/json/InSilicoPredictedImpact

```diff
@@ -0,0 +1,32 @@
+{
+   "$schema": "https://json-schema.org/draft/2020-12/schema",
+   "$id": "https://w3id.org/ga4gh/schema/va-spec/1.0.1/base/json/InSilicoPredictedImpact",
+   "title": "InSilicoPredictedImpact",
+   "maturity": "draft",
+   "type": "object",
+   "description": "The predicted impact of a variant on a transcript and/or protein molecule with respect to a given variant effect predictor.",
```

Comment:
A suggestion for a more generic description ( to also cover regulatory and intergenic regions) - 
A score from an in silico tool used to evaluate likely variant deleteriousness